.github/workflows/miniooni.yml

```diff
@@ -23,6 +23,14 @@ jobs:
         with:
           name: miniooni-darwin-amd64
           path: ./CLI/darwin/amd64/miniooni
+      - uses: actions/upload-artifact@v1
+        with:
+          name: miniooni-linux-armv7
+          path: ./CLI/linux/armv7/miniooni
+      - uses: actions/upload-artifact@v1
+        with:
+          name: miniooni-linux-arm64
+          path: ./CLI/linux/arm64/miniooni
```

Comment:
You're welcome! 
Everything is fine, but shouldn't this change be added to the Linux section?
Thanks.

Reply:
yes, totally! I will indeed move it!